dunwoody_disaster/views/CharacterSelector.py

```diff
@@ -54,11 +54,13 @@ def __init__(self, characters: list[Character]):
         row += 1
 
         container = QWidget()
+        container.setStyleSheet("min-width: 0px;")
```

Comment:
I've found
```suggestion
        container.setMinimumWidth(0)
```
to work better, because then the style won't cascade.

Reply:
I wanted the style to cascade actually, doesn't work as intended the other way.